.circleci/unittest/windows/scripts/install.sh

```diff
@@ -20,4 +25,6 @@ printf "Installing PyTorch with %s\n" "${cudatoolkit}"
 conda install -y -c pytorch-nightly pytorch "${cudatoolkit}"
 
 printf "* Installing torchaudio\n"
-IS_CONDA=true python setup.py develop
+curl --retry 3 https://s3.amazonaws.com/ossci-windows/torchaudio_deps.7z --output /tmp/torchaudio_deps.7z
```

Comment:
> * What does `torchaudio_deps.7z` contain?

The header and the libraries of the third-party dependencies, like `FLAC`, `libmad` and `libmp3lame`.



> * Where is it from?

Built locally by myself.



> * Does it include source or binary?

Binary.



> * How was it built?

Generally speaking, I built them one by one manually according to the steps given by the project itself. However, some of the project support only very old format the MSVC project, so I have to rewrite it using CMake.

> * How to update it if dependencies should change in future?

I hope that I could figure out the automatic build script for those projects. It is not easy because like I said in the previous question that I had to modify them a little bit. But I think it would be fine since some of the dependencies are not updated in a long time, like Sox and libmad. So for those old projects, maybe we could just download deps or maintain a fork. As for the active projects, like `FLAC`, we could just use the steps provided by the project itself.

Reply:
Yeah I'm not entirely sure if I would want to sign us up for this considering any update would have to be done by hand (however infrequent the updates may come).

Is there a way to contribute what we've done to make this dependency archive to those upstream dependencies?